src/main/webapp/style.css

```diff
@@ -8,10 +8,32 @@ hidden {
 }
 
 .container {
-  margin-top: 0px;
+  margin-top: 10px;
   text-align: center;
 }
-
+ul {
+    list-style-type: none;
+}
 .selector-for-some-widget {
   box-sizing: content-box;
 }
+
+#comments {
+  margin-top: 40px;
+}
+
+#comments textarea {
```

Comment:
You can also add a text area with bootstrap that is already formatted and doesn't need any sort of custom styling. Make sure to check out this page: https://getbootstrap.com/docs/4.0/components/forms/